announcer.go

```diff
@@ -119,13 +120,14 @@ func grabAndSendData(prostream string, prostreamPort int, rds string, rdsPort in
 
 //grabCurrentTrackInfo gets the current track info from the CHIRP API
 func grabCurrentTrackInfo(chirpApi string, verbose bool) track {
-	niceUrl := makeNiceUrl(chirpApi)
 
+	niceUrl := makeNiceUrl(chirpApi)
+	
 	writemsg(fmt.Sprintf("About to GET %s", niceUrl), verbose)
-
 	resp, err := http.Get(niceUrl)
+
 	if err != nil {
-		log.Printf("Error connecting to the CHIRP api: , %s\n", err.Error())
+		fmt.Printf("Error connecting to the CHIRP api: , %s\n", err.Error())
```

Comment:
I changed all the `log`s to `fmt` so they go to STDOUT and can get unit tested.

Reply:
I am not a Go expert, but a little research tells me that `log` is safe with concurrent go routines (which `grabAndSendData()` uses) where `fmt` is not. `log` can also be configured to write to STDOUT: `log.SetOutput(os.Stdout)`

That's more about learning, though. It seems like the risk of using `fmt` is where you have so many concurrent writes that the OS starts to interleave them, and I don't think we'll be anywhere near that limit here nor have a problem separating the messages if necessary.

Reply:
Good call. I didn't dig to hard at go. This is an easy change to make.